skin.css

```diff
@@ -2,12 +2,65 @@
     background-color: #ddd;
 }
 
-button.btn {
+.btn {
+  width:170px
   border-radius: 3px;
-  border-style: solid;
-  border-width: 1px;
+  background-color:white;
   font-weight: normal;
-  line-height: 1.15;
-  text-align: center;
-  vertical-align: middle;
+  font-family:sans-serif;
+  padding: 15px;
+  border: 1px; solid lightgrey
+
+
+}
+
+
+  .btn--default{

```

Comment:
I see some indentation and spacing issues here. Sometimes there are two spaces before a classname, sometimes there are no spaces. Sometimes you see `.btn{`, and others `.btn {`. Keep spacing consistent and you keep your teammates happy. :)